src/contexts/addresses.tsx

```diff
@@ -125,6 +159,60 @@ export class Address {
     )
 
     this.transactions.pending = newPendingTransactions
+
+    // Reduce the available balance of the address based on the total amount of pending transactions
+    const pendingSweep = this.transactions.pending.find((tx) => tx.type === 'sweep' || tx.type === 'consolidation')
+    const totalAmountOfPendingTxs = pendingSweep
+      ? this.availableBalance
+      : this.transactions.pending.reduce((acc, tx) => (tx.amount ? acc + BigInt(tx.amount) : acc), BigInt(0))
+    this.availableBalance = this.availableBalance - totalAmountOfPendingTxs
+  }
+
+  async buildSweepTransactions(
+    client: Client,
+    toHash: AddressHash
+  ): Promise<{
```

Comment:
Here we describe the returned type explicitly, but we don't do that for `signAndSendTransaction` below.. or any of the other class' methods actually.

Should we remove this? (the IDE is clever enough in this case, and there are no possible confusion between string / number / bigint like we've seen in the discussion about amounts)